src/telegraf.ts

```diff
@@ -38,9 +38,39 @@ namespace Telegraf {
   }
 
   export interface LaunchOptions {
-    // FIXME: not honored by webhook
     /** List the types of updates you want your bot to receive */
     allowedUpdates?: tt.UpdateType[]
+    /**
+     * **In long polling mode,** when your bot receives one or more updates,
+     * `telegraf` calls the middleware stack and waits for it to complete before
+     * calling `getUpdates` again. This is important: calling `getUpdates`
+     * [confirms the previously received updates to Telegram via the `offset`
+     * parameter](https://core.telegram.org/bots/api#getupdates), but if your
+     * middleware crashes, you may want to request them again, which would
+     * become impossible then.
+     *
+     * The `concurrency` flag (default: `false`) allows you to adjust this
+     * behavior. It determines the maximum number of updates your bot may
+     * already be processing concurrently before making the next `getUpdates`
+     * request. The higher the number, the earlier `telegraf` will make the next
+     * request. **Caution:** setting this value to anything greater than `1` may
+     * cause your bot to miss updates if it crashes!
+     *
+     * **In webhook mode,** it is possible to receive old updates again while
+     * new ones have already been processed, so this is not an issue. The flag
+     * behaves exactly the same way, but its default value is `true`. If you set
+     * it to a lower value, `telegraf` will simply not answer the HTTP requests
+     * until enough of your middleware completed.
+     *
+     * `false` is an alias for `1`. `true` is an alias for `Infinity`. Values
+     * below `1` will be interpreted as `1`.
+     */
+    concurrency?: boolean | number
```

Comment:
It was convenient when I developed the queue module, but as `telegraf.ts` replicates this now, we may as well drop it again.

I found it nice to say `concurrency: false` instead of having to think about which number to pick when avoiding `getUpdates` calls before all middleware has finished.